confgenerator/otel/modular.go

```diff
@@ -24,14 +24,36 @@ import (
 
 const MetricsPort = 20201
 
+type ExporterType int
+
+const (
+	// N.B. Every ExporterType increases the QPS and thus quota
+	// consumption in consumer projects; think hard before adding
+	// another exporter type.
+	OTel ExporterType = iota
+	System
+	GMP
+)
+
+func (t ExporterType) Name() string {
+	if t == System || t == GMP {
+		// OTel and GMP have different exporter types so can share the empty string.
```

Comment:
Does this mean to be `System and GMP have different exporter types`?

Reply:
No, it's referring to the collector's exporter type and not the `ExporterType` enum... I rephrased it.